x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/constants.tsx

```diff
@@ -8,27 +8,26 @@
 import { FormattedMessage } from '@kbn/i18n-react';
 import React from 'react';
 
-export const ACTIVE_COLOR = '#E7664C';
-export const RECOVERED_COLOR = '#54B399';
 export const TOOLTIP_DATE_FORMAT = 'YYYY-MM-DD HH:mm';
+export const ALERT_COUNT_FORMAT = '0.[00]a';
```

Comment:
This format will be applied on alert counts as shown below: ([link](http://numeraljs.com/#format[numeral.js]))
![image](https://user-images.githubusercontent.com/12370520/214331942-5ddf208e-406f-4810-bbcd-f13b58aabc9a.png)